server/service/src/programs/encounter/encounter_updated.rs

```diff
@@ -1,27 +1,23 @@
 use repository::{
-    ClinicianRow, EncounterFilter, EncounterRepository, EncounterRow, EncounterRowRepository,
-    EncounterStatus, EqualFilter, RepositoryError,
+    Document, EncounterFilter, EncounterRepository, EncounterRow, EncounterRowRepository,
+    EncounterStatus, EqualFilter, RepositoryError, StorageConnection,
 };
 use util::uuid::uuid;
 
-use crate::{document::raw_document::RawDocument, service_provider::ServiceContext};
-
 use super::{
     encounter_schema::{self},
     validate_misc::ValidatedSchemaEncounter,
 };
 
 /// Callback called when the document has been updated
 pub(crate) fn update_encounter_row(
-    ctx: &ServiceContext,
+    con: &StorageConnection,
```

Comment:
hmm, are there places where service_context not available where update_encounter_row is used ?

Reply:
I changed it because it was used in sync and there was only a StorageConnection available (but probably no hard reason why it couldn't be a ServiceContext)